dissect/target/tools/fs.py

```diff
@@ -23,9 +25,70 @@
 logging.raiseExceptions = False
 
 
+def human_size(bytes, units=["", "K", "M", "G", "T", "P", "E"]):
+    """Helper function to return the human readable string representation of bytes"""
+    return str(bytes) + units[0] if bytes < 1024 else human_size(bytes >> 10, units[1:])
+
+
 def ls(t, path, args):
```

Comment:
```suggestion
def ls(t, path, args):
```

Since you're changing the whole function anyways, could you add type hints here also ;-)